sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -650,6 +656,7 @@ static struct FloatVect3 compute_accel_from_speed_sp(void)
 
     // Control the airspeed
     sp_accel_b.x = (speed_sp_b_x - airspeed) * gih_params.speed_gain;
+    //BoundAbs(sp_accel_b.x, GUIDANCE_INDI_ACCEL_FWD_BX_LIM);
```

Comment:
If not used, also not needed in airframe files.

Reply:
Agreed